src/serverlessLog.js

```diff
@@ -36,12 +36,12 @@ export function setLog(serverlessLogRef) {
 // logs based on:
 // https://github.com/serverless/serverless/blob/master/lib/classes/CLI.js
 
-function logRoute(method, server, stage, path, maxLength) {
+function logRoute(method, server, path, maxLength) {
   const methodColor = colorMethodMapping.get(method) ?? peachpuff
   const methodFormatted = method.padEnd(maxLength, ' ')
 
   return `${methodColor(methodFormatted)} ${yellow.dim('|')} ${grey.dim(
-    `${server}/${stage}`,
```

Comment:
I would say to maybe leave this in, and just not print it if the option isn't specified? That way people who are cool with the stage change still see the way it was formatted before.

Reply:
Ran into problems after updating serverless-offline to the current version. Basically offline adds the stage name 'development' or 'production' to the Lambda URLs when previous versions did not. Used to be `http://0.0.0.0:3001/query`, now it's `http://0.0.0.0:3001/development/query`. Seems like this PR is almost ready to go. Any chance it could be merged in the next few days?